examples/with-webassembly/pages/api/edge.ts

```diff
@@ -0,0 +1,16 @@
+import type { AddModuleExports } from '../../wasm'
+// @ts-ignore
+import addWasm from '../../add.wasm?module'
+
+const module$ = WebAssembly.instantiate(addWasm)
+
+export default async function handler() {
+  const instance = (await module$) as any
```

Comment:
```suggestion
export default async function handler() {
  const instance = (await WebAssembly.instantiate(addWasm)) as any
```

I'm not aware what the advantage of initiating the wasm module outside the API handler like this is. But I've left it as-is, unless you want to accept this suggestion.